src/main/java/net/dv8tion/jda/core/audio/AudioConnection.java

```diff
@@ -258,28 +266,33 @@ else if (sendHandler == null && sendSystem != null)
 
     private synchronized void setupReceiveSystem()
     {
```

Comment:
The current implementation initializes the receive thread even if there is only an AudioPacketInterceptor, but no ReceiveHandler.

I have tried to cleanup this setup routine as much as I could, but I would appreciate a second pair of eyes checking on it.